L/libint/bundled/patches/shared.patch

```diff
@@ -0,0 +1,13 @@
+diff --git before/libint-2.7.0-beta.5/CMakeLists.txt after/libint-2.7.0-beta.5/CMakeLists.txt
+index 05e6e22..f0aad9a 100644
+--- before/libint-2.7.0-beta.5/CMakeLists.txt
++++ after/libint-2.7.0-beta.5/CMakeLists.txt
+@@ -31,7 +31,7 @@ include(AddCustomTargetSubproject)
+ redefaultable_option(REQUIRE_CXX_API "C++11 Libint interface" ON)
+ redefaultable_option(ENABLE_FORTRAN "Fortran03+ Libint interface" OFF)
+ redefaultable_option(ENABLE_MPFR "Use GNU MPFR library for high-precision testing (EXPERTS ONLY)" OFF)
+-redefaultable_option(LIBINT2_BUILD_SHARED_AND_STATIC_LIBS "Build in one shot static and shared library variants. Uses -fPIC." OFF)
++redefaultable_option(LIBINT2_BUILD_SHARED_AND_STATIC_LIBS "Build in one shot static and shared library variants. Uses -fPIC." ON)
```

Comment:
Can't you call cmake with
```
cmake ... -DLIBINT2_BUILD_SHARED_AND_STATIC_LIBS=ON
```
instead of patching this file?

Reply:
I didn't know I could do that. I'm a cmake novice fumbling my way through this haha.